src/lib.rs

```diff
@@ -490,6 +490,10 @@ struct AttributeData<'input> {
     value: StringStorage<'input>,
     #[cfg(feature = "positions")]
     range: Range<usize>,
+    #[cfg(feature = "positions-extra-attr")]
+    qname_len: u8,
```

Comment:
We can easily use `u16` here. the struct would be padded to 64B anyway. We have plenty of space.